tensorflow_serving/batching/batching_session.h

```diff
@@ -99,6 +99,23 @@ struct BatchingSessionOptions {
   //
   // If left empty, no rounding/padding is performed.
   std::vector<int> allowed_batch_sizes;
+
+  // If set to true, padding is performed for tensors of the same name
+  // but with unequal dimensions, so that all tensors of the same name have
+  // equal dim sizes.
```

Comment:
I now pad with the first element in tensor and support numeric, bool, string and quantized types.